build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
+        classpath 'com.android.tools.build:gradle:3.6.2'
```

Comment:
Bumped the patch version here. I think it may have fixed an issue with the Gradle Daemon getting orphaned but holding a lock on some config files.